custom/icds_reports/static/js/directives/poshan-progress-dashboard/poshan-progress-dashboard.directive.js

```diff
@@ -86,6 +86,9 @@ function PoshanProgressController($scope, $http, $log, $routeParams, $location,
             function (response) {
                 vm.data = response.data;
                 if (vm.step === 'overview' && vm.data) {
+                    if (vm.data['ICDS CAS Coverage']['Number of AWCs Launched']===0) {
```

Comment:
good to make it positive check rather than negative check. Eg:

```
if(num_launched  && num_launched>=1) then is_launched=True
else:
is_launched=False
```

This is for two reasons:
1. Its not so straightforward to have a negative check like notLaunched which makes it hard to read if one wants to use it in positive way or in complex boolean expression
2. We also need to consider case, where value could be null or undefined. in that case location, will be unlaunched.

Reply:
(Sorry for pythonic Sudo code)